wp-content/themes/engage/templates/partial/teammate.twig

```diff
@@ -5,15 +5,16 @@
         </div>
     {% endif %}
     <div class="mate__content">
-      {% if mate.post_type == "team" %}
-        <a class="mate__name" href="/team/{{mate.slug}}">{{ mate.name }}</a>
+      <h3>
+          <a class="mate__name--{{ format }}" {{ format == "preview" ? "href=#{mate.link}" }}>{{ mate.name }}</a>
+      </h3>
+      {% if format == "preview" %}
         {% if mate.getDesignation or mate.vertical %}
           <p class="mate__labels">
               {{ mate.designation }}
           </p>
         {% endif %}
-      {% elseif mate.post_type == "board" %}
-          <p class="mate__name h3">{{ mate.name }}</p>
+      {% elseif format == "full" %}
           <p class="mate__bio">
               {{ mate.content }}
```

Comment:
If this is using `mate.content` it may already be run through the automatic HTML formatter, so wrapping it in a `<p>` may not be the best. May need to change this to a `<div>`. Just take a look at the output and see.